shell/browser/api/electron_api_url_loader.cc

```diff
@@ -577,9 +577,7 @@ void SimpleURLLoaderWrapper::OnResponseStarted(
   dict.Set("statusCode", response_head.headers->response_code());
   dict.Set("statusMessage", response_head.headers->GetStatusText());
   dict.Set("httpVersion", response_head.headers->GetHttpVersion());
-  // Note that |response_head.headers| are filtered by Chromium and should not
-  // be used here.
-  DCHECK(!response_head.raw_response_headers.empty());
+  dict.Set("headers", response_head.headers.get());
```

Comment:
Just curious, did this filtering behavior change? I just want to make sure we're not in a situation where they would be filtered by Chromium again

Reply:
I don't think so—I think the comment here was a bit misguided; I believe the only header that was being filtered out was set-cookie. I checked through most of our test requests and couldn't find anything that was being filtered besides set-cookie, though our tests are not complete coverage.

I suppose we'll find out sooner or later via bug report if I'm wrong!